_shared_content/automate/debug-playbooks.md

```diff
@@ -0,0 +1,21 @@
+# Debug playbooks
+
+## Rate limiting
+
+If too many actions are pushed, it may block running playbooks. In order to address this issue, playbooks have a rate limiting the action runs.
```

Comment:
```suggestion
If too many actions are pushed, it may block running playbooks. In order to address this issue, playbooks have rate limiting applied on their action runs.
```